scripts/cms-any/dispatch-ci.php

```diff
@@ -1,9 +1,10 @@
 <?php
 
-// run on two consecutive days of the week
+// run on three consecutive days of the week
 $dayOfWeek = predictable_random_int('dispatch-ci', 6);
 $nextDayOfWeek = $dayOfWeek === 6 ? 0 : $dayOfWeek + 1;
-$runsOnDaysOfWeek = sprintf('%s,%s', $dayOfWeek, $nextDayOfWeek);
+$nextNextDayOfWeek = $dayOfWeek === 5 ? 0 : ($dayOfWeek === 6 ? 1 : $dayOfWeek + 2);
```

Comment:
This line is really hard to visually parse - a single ternary is okay but chaining them gets confusing.
I'll leave it since it works and this is just for our use, but just a note for next time.